terraform/README.md

```diff
@@ -0,0 +1,14 @@
+# For the bits that are not nixops-able
+
+For now this manages only resources in the main AWS account.
+
+## Usage
+
+Make sure to have the AWS key-pair in the environment, in
+`~/.aws/credentials` or as the EC2 metadata service.
+
+Then run:
+
+```
+nix-shell --run "terraform apply"
```

Comment:
Maybe add a note about that this will present a diff and ask if you want to apply it, and that `apply` won't just go changing things right away.

Reply:
Right, I also forgot the `terraform init` bit

Reply:
fixed